src/pages/popup/send.tsx

```diff
@@ -207,6 +188,12 @@ const Form = ({
     isSubmitting,
     setSubmitting,
   });
+
+  const onChange = (e: React.ChangeEvent<any>) => {
```

Comment:
Curious why you used `onX` here instead of `handleX`?

Reply:
because `handleChange` is provided by the formik props, and this is for the `onChange` prop, so it would have been either:

```ts
<Component onChange={e => ...} />
```
or 
```ts
const onChange = e => ...
<Component onChange={onChange} />
```

Reply:
I understand this now. I've never worked with formik before and had to review the docs with the password bug Kyran was working on. Now I see how `handleChange` is being used. Thanks.